README.md

```diff
@@ -216,7 +216,7 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   projectId: 5000658,
-  quality: 360
+  quality: 360 //optional argument
```

Comment:
in commit message why you do not mention which example is updated? commit message must describe the changes it's made. And please don't use commit reverting or similar things.

Reply:
and please use full stops at the end of the commit message sentence :)

Reply:
you can create a new branch, and do some beautiful commits there, and open pull request :)